src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -84,26 +96,63 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
   } = useVenueEvents({
     venueIds: relatedVenueIds,
   });
-
   const isLoadingSchedule = isLoading || isEventsLoading;
 
   const [selectedDayIndex, setSelectedDayIndex] = useState(0);
 
+  const minDate = useMemo(() => {
+    if (relatedVenueEvents.length) {
```

Comment:
I believe we should we okay without that check, we can add an extra argument to Math.min(..array, SAFE_VALUE) if needed.